modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -222,6 +311,22 @@ class MirrordExecManager(private val service: MirrordProjectService) {
                 throw e
             }
         }
+
+        fun containerStart(): MirrordContainerExecution? {
```

Comment:
Would be nice to avoid this duplication as well if you think it's easy enough.

Reply:
The return type is different, I don't want to go into this right now, also the inclusion of wsl is strange because it can't be wsl and docker at the same time =\